teamworkApp/lib/dbCalls.py

```diff
@@ -0,0 +1,109 @@
+# muddersOnRails()
+# Sara McAllister November 5, 2017
+# Last updated: 11-8-2017
+
+# library for SQLite database calls for teamwork analysis app
+
+import contextlib
+import datetime
+import sqlite3
+
+DB = 'db/development.sqlite3'
+test_DB = 'db/test.sqlite3'
+
```

Comment:
I would like more thorough doc strings, just for future reference back to this file so it's a bit more clear what format the arguments should be in, etc.